bot/exts/events/trivianight/_scoreboard.py

```diff
@@ -123,26 +123,26 @@ def _get_rank(self, member: Member) -> Embed:
         return rank_embed
 
     @discord.ui.button(label="Scoreboard for Speed", style=ButtonStyle.green)
-    async def speed_leaderboard(self, button: Button, interaction: Interaction) -> None:
+    async def speed_leaderboard(self, interaction: Interaction, _: Button) -> None:
         """
         Send an ephemeral message with the speed leaderboard embed.
 
         Parameters:
-            - button: The discord.ui.Button instance representing the `Speed Leaderboard` button.
-            - interaction: The discord.Interaction instance containing information on the interaction between the user
+             - interaction: The discord.Interaction instance containing information on the interaction between the user
             and the button.
+            - button: The discord.ui.Button instance representing the `Speed Leaderboard` button.
         """
```

Comment:
Extra space that shouldn't be there on the `- interaction:` line

Reply:
Yep, fixed :+1: